ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -5,16 +5,19 @@ import {AttributesPage} from 'app/cohort-search/attributes-page/attributes-page.
 import {Demographics} from 'app/cohort-search/demographics/demographics.component';
 import {ListSearchV2} from 'app/cohort-search/list-search-v2/list-search-v2.component';
 import {searchRequestStore} from 'app/cohort-search/search-state.service';
+import {Selection} from 'app/cohort-search/selection-list/selection-list.component';
 import {CriteriaTree} from 'app/cohort-search/tree/tree.component';
 import {domainToTitle, generateId, typeToTitle} from 'app/cohort-search/utils';
 import {Button, Clickable} from 'app/components/buttons';
 import {FlexRowWrap} from 'app/components/flex';
 import {ClrIcon} from 'app/components/icons';
 import {SpinnerOverlay} from 'app/components/spinners';
-import colors, {addOpacity} from 'app/styles/colors';
+import colors, {addOpacity, colorWithWhiteness} from 'app/styles/colors';
 import {reactStyles, ReactWrapperBase} from 'app/utils';
 import {triggerEvent} from 'app/utils/analytics';
+import {currentCohortCriteriaStore} from 'app/utils/navigation';
 import {Criteria, CriteriaType, DomainType, TemporalMention, TemporalTime} from 'generated/fetch';
+import {Messages} from 'primereact/messages';
```

Comment:
Just curious, did you also look at Primereact's [Growl component](https://primefaces.org/primereact/showcase/#/growl)? At a glance, it looks more similar to what the mocks call for, not sure if it's actually a better fit for what we need.

Reply:
Growl might be a better option for this as itdisplay message in overlay vs messages that are supposed to be for inline. I'll change to Growl